### PR TITLE
GDPR IAB | Integration Branch

### PR DESCRIPTION
**Pull request by JoshuaRogan**
Let's merge all GDPR related work into this branch

@Wikia/cake

### CODE REVIEW

File: selenium/test.js [hunk 1]
```diff
@@ -81,16 +81,6 @@ describe("BrowserStack: ", () => {
             assert.equal(cookie.value, cookieState.accepted);
             ensureNoPrompt();
         });
-
-        it("adds the correct cookie when rejected", () => {
```
Comment (JoshuaRogan) [inline selenium/test.js]: We should add a test that disables all of the toggles and verifies that everything is disabled

File: src/ContentManager-test.js [hunk 2]
```diff
@@ -2,15 +2,16 @@ import { assert } from 'chai';
 import ContentManager, { langToContent } from './ContentManager';
 
 describe('ContentManager', () => {
-    it('gives content in the specified language', () => {
-        assert.equal(new ContentManager('de').content.headline, langToContent.de.headline);
-    });
+    // TODO: enable tests once we have new translations CAKE-5023
```
Comment (cschucode) [inline src/ContentManager-test.js]: Do we want to add this test back?

File: src/TrackingOptIn.js [hunk 3]
```diff
@@ -30,17 +30,20 @@ class TrackingOptIn {
...
-            allowedVendors: this.options.enabledVendors,
-            allowedVendorPurposes: this.options.enabledVendorPurposes
+            allowedVendors: enabledVendors,
+            allowedVendorPurposes: enabledPurposes
         });
         this.consentManagementProvider.install();
         this.options.onAcceptTracking();
     };
 
     onRejectTracking = () => {
```
Comment (cschucode) [inline src/TrackingOptIn.js]: I would vote to remove from the code as it remains on GitHub.

File: src/components/App.js [hunk 4]
```diff
@@ -36,83 +43,90 @@ class App extends Component {
...
+        this.props.tracker.trackLearnMoreClick();
+        this.setState({ isScreenOne: false });
+        this.forceUpdate();
+    };
+
+    back = () => {
+        this.props.tracker.trackBackClick();
+        this.setState({ isScreenOne: true });
+        this.forceUpdate();
+    }
```
Comment (cschucode) [inline src/components/App.js]: minor but could use a semi-colon.

File: src/TrackingOptIn.js (hunk 3 above)
Comment (mklucsarits) [reply src/TrackingOptIn.js]: Actually the code is still being used, so I removed the comment completely.

**Comment by mklucsarits** [comment]
@cschucode I addressed your comments and a couple in-person comments from @JoshuaRogan. Merging now.

**Pull request merged**